### PR TITLE
fix some broken links in dev branch2204[skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
fix some broken links in dev branch 2204

### CODE REVIEW

**Comment by nvliyuan** [comment]
fix #4725

**Comment by nvliyuan** [comment]
verified in https://nvliyuan.github.io/yuali-spark-rapids/

**Comment by nvliyuan** [comment]
sometimes (not always) we will get broken links if we don't put link in a single line.
it is recommended to write `[xxx](xxx.md#xxx)` in one single line
rather than `[xx`
`x](xxx.md#xx)`

File: docs/get-started/getting-started-gcp.md [hunk 1]
```diff
@@ -86,8 +86,8 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
 To use notebooks with a Dataproc cluster, click on the cluster name under the Dataproc cluster tab
 and navigate to the "Web Interfaces" tab.  Under "Web Interfaces", click on the JupyterLab or
 Jupyter link to start to use sample [Mortgage ETL on GPU Jupyter
-Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years [Mortgage
-data](https://rapidsai.github.io/demos/datasets/mortgage-data).
+Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years
```
Comment (jlowe) [inline docs/get-started/getting-started-gcp.md:89]: This link is wrapped as described in the PR description as what should be fixed

File: docs/tuning-guide.md [hunk 2]
```diff
@@ -337,7 +337,7 @@ Custom Spark SQL Metrics are available which can help identify performance bottl
 
 Not all metrics are enabled by default. The configuration setting `spark.rapids.sql.metrics.level` can be set
 to `DEBUG`, `MODERATE`, or `ESSENTIAL`, with `MODERATE` being the default value. More information about this
-configuration option is available in the <a href="configs.md#sql.metrics.level">configuration</a> documentation.
+configuration option is available in the [`configuration`](configs.md#sql.metrics.level) documentation.
```
Comment (jlowe) [inline docs/tuning-guide.md:340]: Backticks are usually reserved for cases where we are referring to code or things the user should type verbatim.  This isn't one of those cases.
```suggestion
configuration option is available in the [configuration documentation](configs.md#sql.metrics.level).
```

**Comment by sameerz** [comment]
Good catches!

**Comment by viadea** [comment]
@nvliyuan  I think we should re-target to 22.02 since 22.02 branch is still open. 
Then after that, we should merge it to main and then gh-pages branch.

**Pull request closed without merging**